Changelog.md

```diff
@@ -1,3 +1,7 @@
+## v1.5.4 (2018-12-10)
+
+- Add ability to pass api key to individual API method calls
+
```

Comment:
do we have examples of this documented?

Reply:
@erdey The reason for adding this were issues that we faced when switching between multiple api keys within an internal service, but I think our customers would only use one api key at a time ever anyways.

Reply:
@nothingisfunny 👍 seems fine to leave it undocumented for now then